numpy/f2py/tests/src/array_from_pyobj/wrapmodule.c

```diff
@@ -149,9 +149,9 @@ PyMODINIT_FUNC PyInit_test_array_from_pyobj_ext(void) {
   if (PyErr_Occurred())
     Py_FatalError("can't initialize module wrap (failed to import numpy)");
   d = PyModule_GetDict(m);
-  s = PyString_FromString("This module 'wrap' is auto-generated with f2py (version:2_1330).\nFunctions:\n"
-"  arr = call(type_num,dims,intent,obj)\n"
-".");
+  s = PyUnicode_FromString("This module 'wrap' is auto-generated with f2py (version:2_1330).\nFunctions:\n"
+                           "  arr = call(type_num,dims,intent,obj)\n"
+                           ".");
```

Comment:
Hmm, if this is auto-generated then maybe we should not be editing it?

Reply:
It's in the repo, doesn't go away with `git clean -xdfq` and the content dates back to 2005. I think the string is intended for code generated by `wrapper.c`. However, in that case it is probably best to keep the message in bytes...

Reply:
> I think the string is intended for code generated by 

I think you're right, but the top of this file also says its autogenerated.

Reply:
Probably just left over from before the tests were redone. Note that it is only used in setting up `test_array_from_pyobj.py`, so I think it really doesn't matter.

Reply:
The `wrapmodule.c` is used in `numpy/f2py/tests/test_array_from_pyobj.py` and can be considered constant, that is, it can be manually modified.
For background: he file was generated with f2py as a C/API module template and later modified manually to expose various macro constants to Python for testing..

Reply:
Sounds like that part of the header can just be removed. I'll leave this PR as is with a release note and make a separate PR with the version changes and other small cleanups suggested here.

Reply:
Would you like to wait for this to be merged until the follow-on PR is made?

Reply:
I don't understand your question @mattip, the header is changed already.

Reply:
@mattip No :) I'm waiting for this to be merged. The follow-on PR just changes the version handling.